tools/autograd/templates/Functions.cpp

```diff
@@ -1792,6 +1792,55 @@ Tensor svd_backward(const std::vector<torch::autograd::Variable> &grads, const T
   return u_term + sigma_term + v_term;
 }
 
+Tensor eig_backward(const std::vector<torch::autograd::Variable> &grads, const Tensor& self,
```

Comment:
Do you have a reference containing the gradient formula?
That can come in handy if we come back to this later.

Reply:
I derived it myself, but there is a document that contains similar expression (and I have put a link to it). However, the expression might be slightly different, though.